main.go

```diff
@@ -575,12 +576,17 @@ func auditGitReference(repo *RepoDescriptor, ref *plumbing.Reference) []Leak {
 					} else if to != nil {
 						filePath = to.Path()
```

Comment:
Definitely related, but perhaps beyond the scope of this PR: would it make more sense to default the `filePath` to the `to.Path()`, since that is likely the value expected/wanted by the user since it reflects the current (in the context of the commit) path of the file or the _new_ path in the case where the commit renamed/moved the file? Perhaps I'm missing something or misinterpreting the API, but thought I'd ask.